### ISSUE TITLE
Each time add to cart is done multiple copies of same product is added!!

### ISSUE THREAD

**Issue by elaiyabharathi** [mohsalsaleem/Magic-market#3]
An association of has_one needs to be created so that an user can add one product one time only and an attribute of count needs to be added
![screenshot from 2015-05-24 08 19 30](https://cloud.githubusercontent.com/assets/4281219/7786434/a5ce647e-01ed-11e5-8eb0-92bebdc57bb6.png)

**Comment by mohsalsaleem** [comment]
Yeah.. When I finished this, it was already 2. I wasn't able to think. So I just pushed. What has to be done is, we have to check if the the product is already there for the user and if it is there, we need to have a quantity field and increment it. I just woke up. :-P. Will do now.

**Comment by mohsalsaleem** [comment]
And also, that list won't be present to the admins for taking orders. The model will be orders, which I'll implement today.
